### DISCUSSION TITLE
How do you disable Windows Update step?

### DISCUSSION THREAD

**Discussion by Zageron** [topgrade-rs/topgrade#94] [Q&A] [answered]
Configuration does not have an obvious option to disable it with.

Thanks!

**Comment by s34m** [comment]
Will try to find out if there is a way and else I will add the feature. Also I will have to update the Documentation to add stuff like that.

**Comment by SteveLauC** [comment, marked as answer]
Simply disabling the `system` step should work

**Reply by kris3713** [reply]
Can you show how to do that, please?

**Reply by SteveLauC** [reply]
@FlawlessCasual17 

You can either do it through the CLI:

```sh
$ topgrade --disable system
```

Or you can write it to your configuration file:

```toml
[misc]
disable = ["system"]
```

**Reply by kris3713** [reply]
Thank you so much.
